vm/systemSmartContracts/delegation.go

```diff
@@ -96,6 +96,9 @@ func NewDelegationSystemSC(args ArgsNewDelegation) (*delegation, error) {
 	if args.DelegationSCConfig.MinServiceFee > args.DelegationSCConfig.MaxServiceFee {
 		return nil, fmt.Errorf("%w minServiceFee bigger than maxServiceFee", vm.ErrInvalidDelegationSCConfig)
 	}
+	if args.DelegationSCConfig.MaxServiceFee < 1 {
```

Comment:
maybe add similar condition to args.DelegationSCConfig.MinServiceFee ?

Reply:
min service fee can be 0

Reply:
ok, we can call this with -1 for min fee and 1 for max fee?

Reply:
no. this is an UINT64